Python/ceval_macros.h

```diff
@@ -275,12 +354,12 @@ GETITEM(PyObject *v, Py_ssize_t i) {
 #define UPDATE_MISS_STATS(INSTNAME) ((void)0)
 #endif
 
-#define DEOPT_IF(COND, INSTNAME)                            \
+#define DEOPT_IF(COND, INSTNAME, SIZE)                      \
```

Comment:
As the form of the generated macro and the input macro are further diverging, it might make sense to rename the output macro.